.github/workflows/monitoring-CI.yml

```diff
@@ -71,6 +72,9 @@ jobs:
 
       - name: Upload to Riff Raff
         uses: guardian/actions-riff-raff@v3
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
Does the monitoring release need the NPM_TOKEN if it 's not deployed to npm?

Reply:
Thanks, I just removed this.